frontend/src/metabase/ui/components/inputs/Calendar/Calendar.styled.tsx

```diff
@@ -35,11 +35,11 @@ export const getCalendarOverrides = (): MantineThemeOverride["components"] => ({
           color: theme.fn.themeColor("bg-dark"),
         },
         "&[data-in-range]": {
-          color: theme.fn.themeColor("text-medium"),
+          color: "var(--mb-color-text-hover)",
           borderRadius: 0,
-          backgroundColor: theme.fn.themeColor("brand-lighter"),
+          backgroundColor: "var(--mb-color-background-hover)",
           "&:hover": {
-            backgroundColor: theme.fn.themeColor("brand-lighter"),
+            backgroundColor: "var(--mb-color-background-hover)",
```

Comment:
These lines are the highlight. Uses the semantic color to allow theming.

Reply:
Could you check with Alessio on the changes to foreground text here?